sass/directives/02_base_components/button/_button.scss

```diff
@@ -222,6 +222,101 @@ $massive-button-horizontal-padding: $base-input-font-size * 2;
   }
 }
 
+// Fancy Play Button
+// Initial state
+$ripple-diameter: 24em;
+$play-button-size: 4em;
+$ripple-color: white;
+$ripple-hover-color: #447777;
```

Comment:
I think I chose this color myself. Ask Justin what color he'd like to use from our defined colors